frontend/src/metabase/api/database.ts

```diff
@@ -52,6 +75,10 @@ export const databaseApi = Api.injectEndpoints({
       invalidatesTags: (response, error, { id }) => [
         listTag("database"),
         idTag("database", id),
+        idTag("database-metadata", id),
+        idTag("database-id-fields", id),
+        tag("field"),
+        tag("field-values"),
```

Comment:
We could technically iterate over `response.tables[].fields` and use specific ids here. WDYT?

On the other hand it might be insufficient if we ever want to use `listTag("field")` because technically the list of fields could be affected by this API call.

Reply:
We could, but we would still have to invalidate everything; so there would be no improvement I think